### PR TITLE
Feature/composer update

### PR DESCRIPTION
**Pull request by jiisuominen**
# [UHF-0000](https://helsinkisolutionoffice.atlassian.net/browse/UHF-0000)

## What was done

* Run composer update & restore missing configs.

## How to install

* Make sure your instance is up and running on correct branch.
  * `git checkout UHF-0000_insert_correct_branch`
  * `make fresh`
* Run `make drush-cr`

### CODE REVIEW

File: conf/cmi/core.entity_view_display.node.webform.teaser.yml [hunk 1]
```diff
@@ -26,7 +25,6 @@ content:
 hidden:
   field_form_code: true
   field_lower_content: true
-  field_url_to_form_service: true
```
Comment (teroelonen) [inline conf/cmi/core.entity_view_display.node.webform.teaser.yml:29]: Changes to this file should be reverted.

File: conf/cmi/core.entity_view_display.node.webform.default.yml [hunk 2]
```diff
@@ -43,6 +42,5 @@ content:
     region: content
 hidden:
   field_form_code: true
-  field_url_to_form_service: true
```
Comment (teroelonen) [inline conf/cmi/core.entity_view_display.node.webform.default.yml:46]: Changes to this file should be reverted.

File: conf/cmi/block.block.hdbt_subtheme_breadcrumbs.yml [hunk 3]
```diff
@@ -1,6 +1,6 @@
 uuid: 87173946-5888-43e8-8443-10f7741e700b
 langcode: en
-status: false
+status: true
```
Comment (teroelonen) [inline conf/cmi/block.block.hdbt_subtheme_breadcrumbs.yml:3]: This should be false I think.

**Review by teroelonen — CHANGES_REQUESTED** [review]
Some reverts needed.

File: conf/cmi/block.block.hdbt_subtheme_breadcrumbs.yml (hunk 3 above)
Comment (jiisuominen) [reply conf/cmi/block.block.hdbt_subtheme_breadcrumbs.yml]: done

File: conf/cmi/core.entity_view_display.node.webform.default.yml (hunk 2 above)
Comment (jiisuominen) [reply conf/cmi/core.entity_view_display.node.webform.default.yml]: done.

**Review by teroelonen — APPROVED** [review]
This looks good now 🦖

**Pull request closed without merging**